src/App.tsx

```diff
@@ -10,33 +11,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const handleTabSelect = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
```

Comment:
`handleTabSelect `does exactly the same stuff what `setSelectedTabId` does. Consider using only setSelectedTabId and removing this redundant function
```suggestion
```